packages/react-instantsearch-core/src/connectors/__tests__/connectInfiniteHits.js

```diff
@@ -318,14 +318,14 @@ describe('connectInfiniteHits', () => {
         'theQueryID_1',
         'theQueryID_1',
         // page 2
-        'theQueryID_2',
-        'theQueryID_2',
-        'theQueryID_2',
-        'theQueryID_2',
-        'theQueryID_2',
-        'theQueryID_2',
-        'theQueryID_2',
-        'theQueryID_2',
+        'theQueryID_2_',
+        'theQueryID_2_',
+        'theQueryID_2_',
+        'theQueryID_2_',
+        'theQueryID_2_',
+        'theQueryID_2_',
+        'theQueryID_2_',
+        'theQueryID_2_',
```

Comment:
what's this change for?

Reply:
https://github.com/algolia/react-instantsearch/blob/ad9a17e5beff2794e0d4c4f1d75595ee7f421af4/packages/react-instantsearch-core/src/connectors/__tests__/connectInfiniteHits.js#L265-L274

retrying with a different results updates the hits being rendered (override the cache)